orca-core/src/test/groovy/com/netflix/spinnaker/orca/pipeline/ExecutionLauncherSpec.groovy

```diff
@@ -102,28 +106,11 @@ class PipelineLauncherSpec extends ExecutionLauncherSpec<Pipeline, PipelineLaunc
     launcher.start(json)
 
     then:
+    1 * executionRepository.store(_)
     0 * runner.start(_)
 
     where:
-    config = [pipelineConfigId: "whatever", stages: []]
-    json = objectMapper.writeValueAsString(config)
-  }
-
-  def "does not start pipeline if it does not have a pipeline config id"() {
-    given:
-    startTracker.queueIfNotStarted(*_) >> false
-
-    and:
-    @Subject def launcher = create()
-
-    when:
-    launcher.start(json)
-
-    then:
-    0 * runner.start(_)
-
-    where:
-    config = [stages: []]
+    config = [id: "whatever", stages: [], limitConcurrent: true]
```

Comment:
s/pipelineConfigId/id (I guess we use 'id' instead of pipelineConfigId when kicking off pipelines)